intro.js

```diff
@@ -1780,7 +1782,19 @@
     targetElm.appendChild(overlayLayer);
 
     overlayLayer.onclick = function() {
-      if (self._options.exitOnOverlayClick === true) {
+      if (self._options.overlayClicked === '' || self._options.exitOnOverlayClick === false) {
+        // Do nothing
```

Comment:
This bit is needed for the backwards compatibility as both `overlayClicked` and `exitOnOverlayClick` are in play. I have the `'exit'` default as last if so we can use both together. In the furture `exitOnOverlayClick` should be removed and this code made easier to read.